### PR TITLE
added ds3 and ff4 to deterministic diamond

### PR DESCRIPTION
**Pull request by hlprimm**
Only problem is I don't know how to determine the price of these. I literally copied it from another recipe to not fiddle with syntax. I have no reason for choosing 36. If anyone knows how to calculate a balanced cost, I'd like to hear

### CODE REVIEW

**Comment by ttk2** [comment]
this one will require more discussion, could you make a post in the dev section of the form about it?

**Comment by hlprimm** [comment]
Ok @ttk2 this should be ready to merge now. Just to run down what I did on the commits:

Added ds3 and ff4 to deterministic diamond, ran it by forum. 
Matta suggested to remove both and add only ds3 to random. 
There was confusion over what enchant to lower by .05 to factor in the new ds3, so I lowered FireProt3 by .05 initially. 
After that, matta said re-up FireProt3 to its original percent chance and dock the "highest percentage" enchant, which is FF4, by .05. 
The last commit is just me merging the updates upstream that happened while I let my commit sit on my local machine. 

tl;dr: ds3 has .05 chance in random diamond after lowering ff4 by .05 per matta's suggestion.

For reference:
https://forum.civcraft.co/t/proposal-add-depth-strider-and-feather-falling-to-deterministic-factories/739/19?u=hlprimm

**Comment by ProgrammerDan** [comment]
Looks good, would be best to squash, feel up to it @hlprimm ?

**Comment by hlprimm** [comment]
@ProgrammerDan sure if you have a good guide somewhere x)

**Comment by ProgrammerDan** [comment]
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

Basically follow the steps exactly. Only difference is since you did a master-merge as your last commit, the rebase will ignore that commit -- that's fine. The rebase will set the "branch point" to HEAD of master for you, so that last master-merge will become unnecessary. Rebase is smart enough to know it, and ignore it.

**Comment by hlprimm** [comment]
@ProgrammerDan Looks like it worked? lol

**Comment by ProgrammerDan** [comment]
lol no...

**Comment by ProgrammerDan** [comment]
I'm not even sure how you did that ... huh.

**Comment by hlprimm** [comment]
Lol rip.

**Comment by ProgrammerDan** [comment]
uhhh. Hmm. Yeah this isn't safe to merge, lol. Lemme whack it tonight.

**Comment by hlprimm** [comment]
It said the head of my git repo was ahead of my local machine so I had to force push the squash commit. My pastebin if you're interested x)

**Comment by ProgrammerDan** [comment]
yeah, no worries, I'll demux it.

**Comment by ProgrammerDan** [comment]
did you do ```git push origin master --force``` as a last step? Before doing that, did you issue ```git pull origin master``` or something like it, inbetween the rebase and the push?

**Comment by hlprimm** [comment]
yep the first command only, didn't do the git pull origin master in between!

**Comment by ProgrammerDan** [comment]
Ok, easy fix.

Issue ```git rebase -i HEAD~3```

Then reorder the commits to appear as follows:

<code>
pick f440a3d Remove recipe Craft_Sea_Lanterns
pick 5143188 Squashed commit of the following:
pick 96ebfe1 added ds3 and ff4 to deterministic diamond
</code>

save, let it rebase, then force the push. Should fix things.

**Comment by ProgrammerDan** [comment]
@hlprimm let me know if you need further assistance. The steps outlined corrected the issue on my local pull of your branch -- basically, the rebase followed the merge loop and misordered the commits because of when you branched to make this change -- so a manual reordering to "set things right" is required.

**Comment by hlprimm** [comment]
@ProgrammerDan Hopefully all is good now?

**Comment by ProgrammerDan** [comment]
Did you follow my instructions and force the push? I don't see any change?

**Comment by hlprimm** [comment]
Yep, I ran the rebase to reorder like this:

`pick f471250 Remove recipe Craft_Sea_Lanterns`
`pick 0873f7f Squashed commit of the following:`
`pick d50a6de added ds3 and ff4 to deterministic diamond`

Then the push:

`hlprimm@apollo:~/Code/FactoryMod$ git push -f origin master`
`Username for 'https://github.com': hlprimm`
`Password for 'https://hlprimm@github.com': `
`Counting objects: 7, done.`
`Delta compression using up to 2 threads.`
`Compressing objects: 100% (7/7), done.`
`Writing objects: 100% (7/7), 1.22 KiB | 0 bytes/s, done.`
`Total 7 (delta 4), reused 0 (delta 0)`
`To https://github.com/hlprimm/FactoryMod`
` + 43e5df9...d50a6de master -> master (forced update)`

**Comment by ProgrammerDan** [comment]
Heh, I know that feeling. 

Back to square one on this, locally it looked like it worked but the reorder doesn't seem to have worked.

Let's just abandon this pull (I'm admitting defeat, I hate to see a good pull languish). Checkout Civcraft master, reapply your changes, make a new branch, push it, and make a new pull from that new branch.

To fix your master branch after that, checkout Civcraft master, delete your local "master", then force-push civcraft's master branch as your repo's master branch.

**Comment by hlprimm** [comment]
@ProgrammerDan Ok, will do! Thanks for the help!!
